mods/ra/maps/allies-09a/allies09a-AI.lua

```diff
@@ -0,0 +1,88 @@
+--[[
+   Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+
+WTransWays =
+{
+	{ USSRRFEntry.Location, USSRUnload1.Location },
+	{ USSRRFEntry.Location, USSRUnload2.Location }
+}
+WTransUnits =
+{
+	hard = { { "3tnk", "3tnk", "3tnk", "v2rl", "v2rl" }, { "v2rl", "v2rl", "e4", "e4", "3tnk" } },
+	normal = { { "e1", "e1", "3tnk", "3tnk", "v2rl" }, { "e4", "e4", "e4", "e4", "v2rl" } },
+	easy = { { "e1", "e1", "e1", "e2", "e2" }, { "e2", "e2", "3tnk" } }
+}
+WTransDelays =
+{
+	easy = 9,
+	normal = 7,
+	hard = 5
+}
+BuildDelays =
+{
+	easy = 90,
+	normal = 60,
+	hard = 30
+}
+
+SovietAircraftType = { "yak" }
+Yaks = { }
+
+
```

Comment:
You have several functions with two or three spaces in between them. One space only please.

Reply:
Thank you for your review. Unused space removed. 
[4b4c38e](https://github.com/OpenRA/OpenRA/pull/18695/commits/4b4c38e8bed864fd0e1a1d749bc44fef1960b7b9)